concourse.yml

```diff
@@ -82,6 +98,21 @@ instance_groups:
       - release: concourse
         name: tsa
         properties: {}
+      - release: fisma
+        name: harden
```

Comment:
This style of bosh manifest will expect to have a `properties` for the job even if set to empty hash: `{}` like the `tsa` job above it.